docs/architecture/decoder_service_specification.md

```diff
@@ -69,7 +53,7 @@ required group attributes {
 
 ### Raw message schema
 
-See [Edge Server PubSub Message Schema](edge.md#edge-server-pubsub-message-schema).
```

Comment:
I'd prefer "edge service" to "edge server" as the latter sounds like it's referring to a single machine, although if this is the terminology already used in the docs, I'm fine with keeping it.

Reply:
Sure thing, this would be a change from how things were originally written, but no reason not to do that cleanup now.